ddht/main.py

```diff
@@ -39,6 +35,18 @@ async def main() -> None:
                 f"not under the $XDG_DATA_HOME: {boot_info.base_dir}"
             )
 
+    if args.log_file is None:
+        logdir = log_file = get_xdg_ddht_root() / "logs"
+        logdir.mkdir(parents=True, exist_ok=True)
+
+        log_file = logdir / "ddht.logfile"
```

Comment:
I think that `.log` is a more common extension (and it matches what we're doing in trinity)